### ISSUE TITLE
Can't access my NotionClient: Attribute Error

### ISSUE THREAD

**Issue by felipehlvo** [jamalex/notion-py#279]
**When I run NotionClient("<token_v2>") I get the following error:**

AttributeError                            Traceback (most recent call last)
<ipython-input-8-96d805ae60b2> in <module>
----> 1 run_live_smoke_test("550a49111b3c40dbca5281158de1e2d1b2a2302d1dd8085af3fa18685d87bb3080612a9b917b4203728dd3298dc1697df9cac2d92dd43817b6a88d22db05e1f5ab90ea0ae9cba090f758614c7669", "https://www.notion.so/felipesworkshop/Class-Notes-db77bd678dcb411fa588ff14cc58c787")

<ipython-input-7-aae8bb43887a> in run_live_smoke_test(token_v2, parent_page_url_or_id)
      8 def run_live_smoke_test(token_v2, parent_page_url_or_id):
      9 
---> 10     client = NotionClient(token_v2=token_v2)
     11 
     12     parent_page = client.get_block(parent_page_url_or_id)

~\Anaconda3\lib\site-packages\notion\client.py in __init__(self, token_v2, monitor, start_monitoring, enable_caching, cache_key)
     74             self._monitor = None
     75         if token_v2:
---> 76             self._update_user_info()
     77 
     78     def start_monitoring(self):

~\Anaconda3\lib\site-packages\notion\client.py in _update_user_info(self)
     81     def _update_user_info(self):
     82         records = self.post("loadUserContent", {}).json()["recordMap"]
---> 83         self._store.store_recordmap(records)
     84         self.current_user = self.get_user(list(records["notion_user"].keys())[0])
     85         self.current_space = self.get_space(list(records["space"].keys())[0])

~\Anaconda3\lib\site-packages\notion\store.py in store_recordmap(self, recordmap)
    290     def store_recordmap(self, recordmap):
    291         for table, records in recordmap.items():
--> 292             for id, record in records.items():
    293                 self._update_record(
    294                     table, id, value=record.get("value"), role=record.get("role")

AttributeError: 'NoneType' object has no attribute 'items'

**Comment by nitlev** [comment]
I had the same issue, but solved it by updating the library from `0.0.25` to `0.0.28` (latest)

**Comment by CHBrianLiu** [comment]
Hi there,

I have the same issue here. I ran the same program with same user token and the issue occurred from 2021/02/24. My version of Notion-py is 0.0.25.

![image](https://user-images.githubusercontent.com/38838945/107515866-95836b00-6be6-11eb-8136-2e2a43ad33c3.png)

```
Result: Failure
Exception: AttributeError: 'NoneType' object has no attribute 'items'
Stack:   File "/azure-functions-host/workers/python/3.8/LINUX/X64/azure_functions_worker/dispatcher.py", line 355, in _handle__invocation_request
    call_result = await self._loop.run_in_executor(
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/azure-functions-host/workers/python/3.8/LINUX/X64/azure_functions_worker/dispatcher.py", line 542, in __run_sync_func
    return func(**params)
  File "/home/site/wwwroot/DailyWorkJournal/__init__.py", line 28, in main
    client = NotionClient(token_v2=os.environ["USER_TOKEN"])
  File "/home/site/wwwroot/.python_packages/lib/site-packages/notion/client.py", line 68, in __init__
    self._update_user_info()
  File "/home/site/wwwroot/.python_packages/lib/site-packages/notion/client.py", line 75, in _update_user_info
    self._store.store_recordmap(records)
  File "/home/site/wwwroot/.python_packages/lib/site-packages/notion/store.py", line 292, in store_recordmap
    for id, record in records.items():
```

**Comment by felipehlvo** [comment]
> I had the same issue, but solved it by updating the library from `0.0.25` to `0.0.28` (latest)

This worked, thanks!
